src/screens/FeedScreen/index.js

```diff
@@ -122,12 +122,14 @@ const FeedScreen = (props) => {
     getDataFeedsHandle(postOffset, false, nextTargetFeed);
   };
 
-  const refreshMoreText = (index) => {
+  const refreshMoreText = (index, haveSeeMore) => {
     setUpdateIndex(index);
-    setUpdateMoreText(true);
-    setTimeout(() => {
-      setUpdateMoreText(false);
-    }, 1);
+    if (haveSeeMore) {
+      setUpdateMoreText(true);
+      setTimeout(() => {
+        setUpdateMoreText(false);
+      }, 1);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setTimeout` function is used to set `updateMoreText` back to false after 1 millisecond. This is a very short time and might not be noticeable by the user. If the purpose of this code is to show some loading or updating text, you might want to consider increasing the timeout duration.
```diff
-      }, 1);
+      }, 1000); // Change this value based on your requirements
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->